### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by bxin**
Based on discussions at weekly AOS meetings and other discussions with Doug Neill.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -48,14 +48,261 @@
...
+
+The M2 LUT actuator weight component was supplied by the M2 vendor Harris. The Rubin team plans to recalculate these forces to crosscheck the Harris results.
+
+The M2 and the camera hexapods do not have actuator weight components in their LUTs because the output of the hexapod LUTs are positions instead of forces.
+
+#################
+Static components
+#################
+
+The static component of the LUT doesn't vary with external conditions. For the mirrors, these are the forces that are needed to bend out the low spatiall frequency factory figuring error. These were supplied by the vendors during factory acceptance testings. We will not change these components during commissioning and operations, unless somehow it can be proven that the figuring errors are different from what were determined at the factories.
```
Comment (teweitsai) [inline index.rst]: spatiall --> spatial

File: index.rst [hunk 2]
```diff
@@ -48,14 +48,261 @@
...
+- LSSTCam normal operation mode (using four corner wavefront sensors).
+
+#########################
+Appendix A - Useful links
+#########################
+
+M1M3
+####
+
+gravity:
```
Comment (teweitsai) [inline index.rst]: gravity --> Gravity

File: index.rst [hunk 3]
```diff
@@ -48,14 +48,261 @@
...
+- https://github.com/lsst-ts/ts_m1m3support/blob/master/SettingFiles/Tables/AzimuthYTable.csv
+- https://github.com/lsst-ts/ts_m1m3support/blob/master/SettingFiles/Tables/AzimuthZTable.csv
+
+Thermal (place holder for now):
+
+- https://github.com/lsst-ts/ts_m1m3support/blob/master/SettingFiles/Tables/ThermalXTable.csv
+- https://github.com/lsst-ts/ts_m1m3support/blob/master/SettingFiles/Tables/ThermalYTable.csv
+- https://github.com/lsst-ts/ts_m1m3support/blob/master/SettingFiles/Tables/ThermalZTable.csv
+
+static:
```
Comment (teweitsai) [inline index.rst]: static --> Static

**Comment by sandrinethomas** [comment]
Should we add a discussion on the origin of the elevation angle for each of the components?

File: index.rst [hunk 4]
```diff
@@ -48,14 +48,261 @@
...
+
+The gravity components of the LUTs have different elevation ranges,
+
+- [-8.5, 98.5] deg for M1M3. Occasionally M1M3 may go out of the [0, 90] deg range, and we want to make sure the glass mirror can still be supported properly. Hence the 8.5 deg margin on both sides.
+- [-270, 90] deg for M2. In normal operations, M2 elevation will never go beyond the [0, 90] deg range. But M2 has a cart. For testing and engineering purposes, it needs to be able to be rotated by 360 deg. Beyond the [0, 90] deg range, glass safety is still the concern, but not image quality.
+- [0, 90] deg for the M2 and camera hexapods. The hexapods may go out of the [0, 90] deg range. But we won't be imaging outside of that range. Unlike the mirrors, where outside of [0, 90] deg proper LUT forces are still needed to counter gravity, for hexapods, we do not have to compensate position offset due to gravity. If the elevation angle goes smaller than 0 deg, we will use LUT values for 0 deg. If it goes larger than 90 deg we will use LUT values for 90 deg.
+
```
Comment (sandrinethomas) [inline index.rst:120]: Should we add here the origin of the elevation angle value for each components?

File: index.rst [hunk 5]
```diff
@@ -48,14 +48,261 @@
...
+- Nth order standard polynomials of the :math:`cosine` of the elevation angle (:math:`\cos(\theta_e)`),
+- Fourier series up to N cycles in the [0, 90] deg range, and
+- piecewise linear interpolation with input data at 5 deg increment.
+
+In choosing which functional form to use, our main considerations are:
+
+#. Consistency between the LUTs. If the needs of each component is drastically different, it is OK to use different functional forms for them. However, unless there are good reasons for doing so, we prefer to keep the functional form consistent between the LUTs. This will make future work for fine-tuning the LUTs using real measurements and updating the LUTs much more straightforward and less error-prone.
+#. Simplicity. There may be many functional forms that can meet our accuracy requirements. Here the criteria is to make sure the error due to finite number of coefficients is much less than the non-repeating error of the corresponding degree of freedom (DOF). Document-36395 :cite:`Document-36395` shows an analysis that demostrates that the 5th order standard polynomials are accurate enough in all cases. And it is the simplest form among the above.
```
Comment (sandrinethomas) [inline index.rst]: Typo near the end of the paragraph: demonstrate

File: index.rst [hunk 6]
```diff
@@ -48,14 +48,261 @@
...
+
+#################
+Static components
+#################
+
+The static component of the LUT doesn't vary with external conditions. For the mirrors, these are the forces that are needed to bend out the low spatiall frequency factory figuring error. These were supplied by the vendors during factory acceptance testings. We will not change these components during commissioning and operations, unless somehow it can be proven that the figuring errors are different from what were determined at the factories.
+
+As for the hexapods, the :math:`C_0` defined in Sec. :ref:`sec-p5` is the static component. For now, all six coefficients for the 5th order standard polynomial for each hexapod have been determined using results from FEA analyses. Once we have the hexapods mounted on the telescope mount assembly (TMA), we will use Laser Trackers (LTs) to calibrate the LUTs for both hexapods. It is expected that the calibrated values of :math:`C_0` will be quite different from the FEA values, because they depend on the actual installations of the hexapods on the TMA.
+
```
Comment (sandrinethomas) [inline index.rst:181]: At what temperature do we measure C0 for the hexapods?

File: index.rst (hunk 4 above)
Comment (bxin) [reply index.rst:120]: Can you say more about what you mean by "origin"? The elevation angle is uniquely determined once we know the optical axis orientation and the gravity vector. It is not coordinate system specific, right?

File: index.rst (hunk 6 above)
Comment (bxin) [reply index.rst:181]: Very good point. This needs to be defined at a reference temperature (Tref). I believe we need to use the same Tref for all subsystems. Tref=21C seems to be a reasonable choice (This is what Harris has been using for M2). I will edit accordingly.

File: index.rst (hunk 4 above)
Comment (sandrinethomas) [reply index.rst:120]: I meant to define where the information of the elevation comes from (TMA or internal inclinometer).

**Review by sandrinethomas — APPROVED** [review]
This tech note is ready to be publish and serve as the standard used for LUT.
A change in the standard will then require to have a discussion with Bo, Doug, Chuck and Sandrine at a minimum

**Review by cclaver — APPROVED** [review]
This pull request is ready to merge on to the master branch.  Others can sync heir local masters and branch accordingly for further edits.

File: index.rst (hunk 4 above)
Comment (bxin) [reply index.rst:120]: Thanks. See the new paragraph I added to the end of Sec. 3.

**Comment by cclaver** [comment]
First draft is in acceptable for to sync with the master branch.  Further edits will be conducted on additional branches.

**Pull request merged**